src/west/commands/project.py

```diff
@@ -38,16 +38,64 @@ def __init__(self):
             '''))
 
     def do_add_parser(self, parser_adder):
-        return _add_parser(parser_adder, self, _project_list_arg)
+        default_fmt = '{name:14} {path:18} {revision:13} {url} {cloned}'
+        fmt_epilog = textwrap.dedent('''\
+        FORMAT STRINGS
+
+        Projects are listed using a Python 3 format string. Arguments
+        to the format string are accessed by name.
+
+        The default format string is:
+
+        "{}"
+
+        The following arguments are available:
+
+        - name: the name of the project in the manifest
+        - url: the remote full URL of the project as specified by the manifest
+        - path: the relative path to the project as given in the manifest
+          (or relative to the top level directory, if --meta is given)
+        - abspath: an absolute and normalized path to the project as it
+          is cloned on the local file system
+        - revision: the manifest revision of the project
+        - cloned: the string "(cloned)" if the project has been cloned,
```

Comment:
just thinking out loud here: could status be a better term ?
and then status can be: `cloned`, `not cloned` and at the same extendable with other statuses in future.
comments / other thoughts ?